packages/client/src/pages/Safe.js

```diff
@@ -134,7 +135,11 @@ function Safe({ web3 }) {
   };
 
   const onConfirmAction = async ({ uuid }) => {
-    await executeAction(uuid).catch((error) => showErrorModal(error));
+    const events = await executeAction(uuid).catch((error) => showErrorModal(error));
+    if (events) {
+      const actions = events.filter(e => e.type.endsWith("ActionExecuted"));
```

Comment:
Do we have a defined list of all action types? If so we should put them in enum and use by reference

Reply:
We have them defined in contracts, but there are a lot of them, and when every transaction is executed it can trigger a couple of events, we are currently interested only in two types of actions: TransferTokenToAccountActionExecuted and  TransferNFTToAccountActionCreated. In the future this might be a bigger list of actions, if we create a success screen for other transactions.

Reply:
My PR removes all the actions in TreasuryActions, so we will only need to ingest `ActionExecuted` event, which will have the action name + params attached.

Reply:
Let's create a ACTION_TYPES enum in enums.js and just put those two types there for right now
```
ACTION_TYPES = {
     TRANSFER_TOKEN_TO_ACCOUNT_ACTION_EXECUTED = "TransferTokenToAccountActionExecuted"
     TRANSFER_NFT_TO_ACCOUNT_ACTION_EXECUTED = "TransferNFTToAccountActionCreated"
}

```
and use it like this:
```suggestion
      const actions = events.filter(e => e.type == TRANSFER_TOKEN_TO_ACCOUNT_ACTION_EXECUTED);
```

Reply:
I'll update this once Jackson's PR gets merged

Reply:
Updated!